File: Backend/MovieExtended/MovieExtendedApi/Domain/Models/Entities/Language.cs
namespace Domain.Models.Entities
{
    public class Language
    {
        public Language(int id, string name, File trackFile , File subtitles)
        {
            Id = id;
            Name = name;
            TrackFile = trackFile;
            Subtitles = subtitles;
        }

        public Language(string name,  File trackFile , File subtitles)
        {
            Name = name;
            TrackFile = trackFile;
            Subtitles = subtitles;
        }

        protected Language()
        {
        }

        public virtual int Id { get; protected set; }

        public virtual string Name { get; protected set; }

        public virtual File TrackFile { get; set; }
        
        public virtual File Subtitles { get; set; }


    }
}

File: Android/app/src/main/java/com/lod/movie_extended/data/model/Language.java
package com.lod.movie_extended.data.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Жамбыл on 09.01.2016.
 */
public class Language {

    @SerializedName("Id")
    private int id;
    @SerializedName("Name")
    private String name;
    @SerializedName("TrackFile")
    private TrackFile trackFile;
    @SerializedName("Subtitles")
    private Subtitle subtitle;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public TrackFile getTrackFile() {
        return trackFile;
    }

    public Subtitle getSubtitle() {
        return subtitle;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setTrackFile(TrackFile trackFile) {
        this.trackFile = trackFile;
    }

    public void setSubtitle(Subtitle subtitle) {
        this.subtitle = subtitle;
    }

    public static class TrackFile {
        @SerializedName("Id")
        private int id;

        public TrackFile(int id) {
            this.id = id;
        }

        public int getId() {
            return id;
        }
    }

    public static class Subtitle {
        @SerializedName("Id")
        private int id;

        public Subtitle(int id) {
            this.id = id;
        }

        public int getId() {
            return id;
        }
    }
}
